packages/vue-client/vite.config.ts

```diff
@@ -12,4 +12,14 @@ export default defineConfig({
     },
     preserveSymlinks: true, // defaults to false, but with default imports from @ofgcommunity/variants-shared in componentes don't work
   },
+  server: {
+    proxy: {
+      "/api": {
```

Comment:
Remind me what this does again?

Is this just for development so I can visit `localhost:3000/api/games` instead of `localhost:3001/games`?

Reply:
I don't understand everything that's happening there under the hood, but this makes the Vite server running on `localhost:3000` redirect calls of `/api/xyz` to `localhost:3001/xyz/` and thereby resolves issues with CORS during development where the client and the server/api are not served by the same origin (different port).

But I just realized that this means the API would have to listen on the path `/api/` in productive mode, while currently it's listening on `/`. Currently http://www.govariants.com/game/63137d6a6a5edf37a48028fe returns (Vue-) HTML and http://www.govariants.com/games/63137d6a6a5edf37a48028fe returns JSON. :monocle_face:

Reply:
I was wrong about `/api/` in production mode. The client creates different request URLs in production mode, see:
```
export const SERVER_URL =
  process.env.NODE_ENV === "production" ? window.location.origin : "/api";
```

So a change isn't necessary, but it is preferrable to avoid collisions and to have the same URLs during development and production. I'll do another PR for that.